src/pages/dev/send-tokens/roadmap.mdx

```diff
@@ -0,0 +1,25 @@
+# Interchain Token Service Roadmap
+
+## Background
+
+The [Interchain Token Service (ITS)](/dev/send-tokens/introduction) was released to make custom ERC-20 token transfers easier across different blockchains. Unlike bridged tokens, Interchain Tokens are sent between chains via [General Message Passing (GMP)](/dev/general-message-passing/overview) rather than minted on the Axelar chain itself.
+
+The [`InterchainTokenService`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/InterchainTokenService.sol) contract deploys a [`TokenManager`](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/token-manager/TokenManager.sol) to handle multichain token operations such as minting, burning, and locking for every chain that a token is registered to. Tokens are tracked through a unique `interchainTokenId` that persists across these chains.
+
+### Types of Interchain Tokens
+
+There are three types of Interchain Token integrations:
+
+- [Canonical Interchain Tokens](/dev/send-tokens/glossary#canonical-interchain-token) — Interchain Tokens that exist on the chain they were originally deployed on. The Token Manager duplicates this token by deploying it on other chains, accepting the original as the standard across all chains. 
```

Comment:
Is a canonical token a subset of a standardized token? If so, we should make this clear on the glossary as well.

Reply:
Yes it is